libs/crypto/mbedtls/config.h

```diff
@@ -118,6 +118,11 @@
 #define MBEDTLS_PLATFORM_CALLOC_MACRO(X,Y) jsvMalloc((X)*(Y))
 #define MBEDTLS_PLATFORM_FREE_MACRO(X) jsvFree(X)
 
+/* use flash safe version of memcpy so that rodata can be moved to irom */
+#ifdef ESP8266
+#define memcpy(d,s,n) flash_memcpy(d,s,n)
```

Comment:
Shame it's needed really, but it's good that you've managed to avoid modifying mbedTLS.